hasher-matcher-actioner/hmalib/common/timebucketizer.py

```diff
@@ -174,8 +178,53 @@ def get_records(
 
         for file in file_list:
             with open(file, "r") as my_file:
-                content_list.extend(
-                    list(map(type_class().from_csv, csv.reader(my_file)))
-                )
+                content_list.extend(list(map(type_class.from_csv, csv.reader(my_file))))
 
         return content_list
+
+    @staticmethod
+    def squash_content(
+        location: datetime.datetime,
```

Comment:
What is location? Do not remember discussing this. Can you add an explanation to the docstring?

Reply:
My memory (and I may be making stuff up here) was that we keep going back in time until we find a squashed bucket. We wanted to go back at most 1 day so that we don't keep looping infinitely.

Reply:
Oh I thought our plan was to take in a location value, and we squash all records from that point till 1 day before / last squash. I thought this way if they wanted it for now it'd be easy to just pass datetime.now()